lib/google_scraping_web/templates/layout/_header.html.heex

```diff
@@ -0,0 +1,7 @@
+<header class="mb-5">
+  <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
+    <div class="container-fluid">
+    <a class="navbar-brand" href="/">Google Scraping</a>
+    </div>
```

Comment:
Small indentation comment

```suggestion
    <div class="container-fluid">
      <a class="navbar-brand" href="/">Google Scraping</a>
    </div>
```

Reply:
@bterone thanks! Is there any formatter for `.heex` files? I tried to find something, but no luck. What you are using for it?